src/Apps/Artwork/Components/Share.tsx

```diff
@@ -54,33 +73,38 @@ export class Share extends React.Component<ShareProps> {
 
   render() {
     return (
-      <Container flexDirection="column" p={2}>
-        <Sans size="3" weight="medium" color="black100" mb={2}>
-          Share
-        </Sans>
-        <Flex flexDirection="row" mb={1}>
-          <SansGrow size="2" color="black60" mr={4}>
-            <URLInput
-              type="text"
-              readOnly
-              value={this.props.url}
-              innerRef={input => (this.input = input)}
-              onClick={this.selectURL}
-            />
-          </SansGrow>
-          <a onClick={this.selectURL}>
+      <Container>
+        <CloseIconWrapper>
```

Comment:
At this point we should avoid manually defining pixel values and instead defer to our [palette / styled-system theme](https://github.com/artsy/palette/blob/master/src/Theme.tsx#L122). This can be simplified to 

```jsx
<Box top={1} right={1}>
  <CloseIcon name="close" onClick={this.props.onClose} />
</Box>
```